### PR TITLE
finished the quicksort code challenge

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/quickSort/QuickSort.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
+package code401.challenges.quickSort;
+
+import java.util.Arrays;
+
+public class QuickSort {
+
+    public void quickSortFunc(int[] arr, int left, int right)
+    {
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/quickSort/QuickSort.java:8]: In Java it is standard practice to put your opening `{` on the same line as the method definition/conditional/loop

File: code401-challenges/src/main/java/code401/challenges/quickSort/QuickSort.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        test.quickSortFunc(arr, 0, right-1);
+        System.out.println(Arrays.toString(arr));
+    }
+
+
+
+
+
+//
+//    void sort(int arr[], int low, int high)
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/quickSort/QuickSort.java:63]: CORPSE CODE!
